transport/TransportImpl.cpp

```diff
@@ -1050,7 +993,14 @@ void TransportImpl::onIceTcpConnect(std::shared_ptr<Endpoint> endpoint,
 {
     if (_rtpIceSession && _isRunning)
     {
-        if (!_jobQueue.addJob<TcpConnected>(*this, endpoint, source, std::move(packet)))
+        const bool posted = _jobQueue.post(_jobCounter,
+            utils::bind(&TransportImpl::internalIceTcpConnect,
+                this,
+                std::cref(endpoint),
+                std::cref(target),
+                utils::rvalueParam(std::move(packet))));
```

Comment:
Need use `utils::rvalueParam` to pass packets are copy constructor is deleted

Reply:
hmm, that is unfortunate. I wonder if there could be a utils::moveXX for this to avoid the stacking.